.github/pull_request_template.md

```diff
@@ -9,25 +12,25 @@ Before you create this PR confirm that it meets all requirements listed below by
 - [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
 - [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors and are passing (See [Contributor Guide]).
 - [ ] My PR does not decrease the code coverage, or I have __a very special case__ and explained on the PR description why this PR decreases the coverage.
-- [ ] I updated/added relevant documentation (doc comments with `///`) and updated/added examples in `doc/examples`.
-- [ ] I have formatted my code with `flutter format` and the `flutter analyze` does not report any problems.
+- [ ] My PR title starts with a [Conventional Commit] type (`fix:`, `feat:`, `doc:` etc).
```

Comment:
```suggestion
- [ ] The title of my PR starts with a [Conventional Commit] prefix (`fix:`, `feat:`, `doc:` etc).
```
Previous items have "my PR" as a subject, so better to put forward the word **title** here.